pkg/Library/CoreXenon/Designer/GraphService.js

```diff
@@ -259,8 +259,10 @@ const constructCandidates = (atomName, {types, inputs}, systemCandidates) => {
 };
 
 const chooseCandidatesForType = (type, candidates) => {
+  const types = type.split('|');
+  const matches = candidateType => types.some(type => TypeMatcher(type, candidateType));
```

Comment:
TypeMatcher is supposed to handle this

Reply:
ops, i used the wrong delimiter in the NodeTypes file, that's why it didn't work, fixed now